regcore/migrations/0002_auto_20160308_2228.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import mptt.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('regcore', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='regulation',
+            name='children',
+        ),
```

Comment:
This is destructive. To migrate data, I think we'll need to

1. Add all new fields
2. Delete all non-root models
3. For each root model, deserialize and reinsert children
4. Remove the `children` field